src/tests/DomainHost.t.sol

```diff
@@ -182,9 +180,17 @@ contract DomainHostTest is DssTest {
     function testAuth() public {
         host.deny(address(this));
 
-        bytes[] memory funcs = new bytes[](2);
-        funcs[0] = abi.encodeWithSelector(EmptyDomainHost.lift.selector, 0);
-        funcs[1] = abi.encodeWithSelector(EmptyDomainHost.rectify.selector);
+        bytes[] memory funcs = new bytes[](9);
+        funcs[0] = abi.encodeWithSelector(DomainHost.rely.selector, address(0));
+        funcs[1] = abi.encodeWithSelector(DomainHost.deny.selector, address(0));
+        funcs[2] = abi.encodeWithSelector(DomainHost.file.selector, bytes32(""), address(0));
```

Comment:
No need to test these as it is checked in `checkAuth` and `checkFileAddress`.

Reply:
Removed!